fluidBuild.config.cjs

```diff
@@ -23,7 +23,14 @@ module.exports = {
 			script: false,
 		},
 		"compile": {
-			dependsOn: ["commonjs", "build:esnext", "build:test", "build:copy"],
+			dependsOn: [
+				"commonjs",
+				"build:esnext",
+				"build:test",
+				"build:copy",
+				// "^build:rename-types",
```

Comment:
Leaving the same comment here for completion: let's make sure this gets cleaned up 😋